pkgs/by-name/je/jetuml/package.nix

```
@@ -0,0 +1,67 @@
+{
+  stdenvNoCC,
+  lib,
+  makeDesktopItem,
+  fetchurl,
+  jdk,
+  makeWrapper,
+}:
+
+let
+  jdkWithFX = jdk.override { enableJavaFX = true; };
+in
+stdenvNoCC.mkDerivation rec {
+
+  pname = "jetuml";
+  version = "3.9";
+
+  src = fetchurl {
+    url = "https://github.com/prmr/JetUML/releases/download/v${version}/JetUML-${version}.jar";
```

Should use `finalAttrs` rather than `rec` as it integrated with `callPackage`'s override fixpoint.
```suggestion
stdenvNoCC.mkDerivation (finalAttrs: {

  pname = "jetuml";
  version = "3.9";

  src = fetchurl {
    url = "https://github.com/prmr/JetUML/releases/download/v${finalAttrs.version}/JetUML-${finalAttrs.version}.jar";
```